doc/classes/RoomGroup.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="RoomGroup" inherits="Spatial" version="3.4">
+	<brief_description>
+		Groups [Room]s together to allow common functionality.
+	</brief_description>
+	<description>
+		Although [Room] behaviour can be specified individually, sometimes it is faster and more convenient to write functionality for a group of rooms.
+		[RoomGroup]s should be placed as children of the [b]room list[/b] (the parent [Node] of your [Room]s), and [Room]s should be placed in turn as children of a [RoomGroup] in order to assign them to the RoomGroup.
+		A [RoomGroup] can for example be used to specify [Room]s that are [b]outside[/b], and switch on or off a directional light, sky, or rain effect as the player enters / exits the area.
+		[RoomGroup]s receive [b]gameplay callbacks[/b] when the [code]gameplay_monitor[/code] is switched on, as [code]signal[/code]s or [code]notification[/code]s as they enter and exit the [b]gameplay area[/b] (see [RoomManager] for details).
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+	</methods>
+	<members>
+		<member name="roomgroup_priority" type="int" setter="set_roomgroup_priority" getter="get_roomgroup_priority" default="0">
```

Comment:
Should this be `room_group_priority` to properly snakecaseify `RoomGroup`?

Reply:
Ah yes I do remember spending ages trying to decide whether roomgroup and roomlist should be one word or separated by convention. I can change it. I think I was a bit worried that by separating them it wasn't as clear that they are a concept within the engine, rather than literally, a list of rooms, or a group of rooms.

Reply:
Yeah I'm on the fence too, it does seem somewhat clearer as `roomgroup` even if that doesn't follow proper conversion rules. Let's keep it as is then.

Reply:
After discussing we currently think keeping this as one word may be better (if a little quirky).